### PR TITLE
Adding script to calculate BLEU, METEOR, CIDER, SPICE, ROUGE metrics

### PR DESCRIPTION
**Pull request by vedanuj**
Adding a script to calculate BLEU, METEOR, CIDER, SPICE, ROUGE metrics. This is based on a [fork](https://github.com/flauted/coco-caption.git@python23) of the original library for [coco-caption](https://github.com/tylin/coco-caption). This works with python 3 and all the dependent libraries can be installed using pip.

Command to run the script : 

```
python pythia/scripts/eval.py --reference_json <path> --predicted_json <path>
```

### CODE REVIEW

**Comment by vedanuj** [comment]
@apsdehal @endernewton @meetshah1995 please review

File: pythia/scripts/eval.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```
Comment (apsdehal) [inline pythia/scripts/eval.py]: Change the name of the script to `coco_caption_eval.py` and put it under `pythia/scripts/coco/`

File: pythia/scripts/eval.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                d["image_id"] = c["imgid"]
+                img_ids.append(c["imgid"])
+                d["caption"] = c["raw"]
+                references.append(d)
+    img_ids = list(set(img_ids))
+
+    with open(args.predicted_json, "r") as f:
+        preds = json.load(f)
+
+    datasetGTS = {"annotations": references}
```
Comment (apsdehal) [inline pythia/scripts/eval.py]: Let's not use camelCase styling in our code. `datasets_gts`. For cocoeval it is fine as it is an external library but we shouldn't follow that style in our code.

**Comment by vedanuj** [comment]
Made requested changes. 

```
python pythia/scripts/coco/coco_caption_eval.py --reference_json <path> --predicted_json <path>
```

**Pull request merged**